hl7-mllp/2.1/modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,35 @@
+= HL7 MLLP Connector 2.1 - Mule 4
+:page-aliases: connectors::hl7/hl7-mllp-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+Anypoint Connector HL7 Minimal Lower Layer Protocol (MLLP) Connector (HL7 MLLP) provides connectivity and parsing functionality for HL7 MLLP messages.
```

Comment:
Clarify the exact full connector and the abbreviated connector name; I am guessing...:

Anypoint Connector for HL7 Minimal Lower-Layer Protocol (MLLP) (HL7 MLLP Connector)

Reply:
Be sure these names are adhered to throughout.

Reply:
check